src/sub-blocks/BasicCard/BasicCard.tsx

```diff
@@ -23,33 +22,46 @@ const BasicCard = (props: BasicCardProps) => {
         list,
         buttons,
         iconPosition = IconPosition.Top,
+        controlPosition = 'content',
         ...cardParams
     } = props;
     const titleId = useUniqId();
     const descriptionId = useUniqId();
+    const areControlsInFooter = controlPosition === 'footer';
 
     return (
         <CardBase
             className={b()}
             {...cardParams}
             extraProps={{'aria-describedby': descriptionId, 'aria-labelledby': titleId}}
         >
-            <CardBase.Content>
+            <CardBase.Content key="content">
```

Comment:
why do we need to add a constant key?

Reply:
There is no necessity. Will remove.
In the process of development, before CardBase refactoring, it must be added to a child because I passed CardBase children as array of elements. Thanks for the notice.